python/mxnet/ndarray/ndarray.py

```diff
@@ -607,7 +610,7 @@ def __getitem__(self, key):
                [[4., 5.],
                 [6., 7.]]], dtype=float32)
         >>> x[[0, 1], :].asnumpy()  # equivalent
-        array([[[0., 1.],
+        array([[[0., 1.],g[]
```

Comment:
Thanks for your contribution! `g[]` seems to be unrelated.

Reply:
Thanks! Will fix soon.